scripts/setup-ubuntu.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+TARGET=$1
+
+# TODO: Assert TARGET is one of "trusty", and "xenial"
+
+echo "[Install] dependencies"
+sudo apt-get update
+
+sudo apt-get install -y linux-image-extra-$(uname -r)
+sudo apt-get install -y git curl wget
+
+echo "[Install] docker-compose"
+sudo sh -c 'curl -sL https://github.com/docker/compose/releases/download/1.9.0/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose'
```

Comment:
Make sure we avoid downloading `docker-compose` for 2nd time if binary already exists.